index.js

```diff
@@ -68,15 +68,25 @@ const rudy = async (req, res, next, logging) => {
   })
 }
 
+const slowloris = (HTTPServer) => {
+  HTTPServer.headersTimeout = 1000
```

Comment:
Maybe in the future option to set this via config?

Reply:
Creating an issue for that